### PR TITLE
[PATCH v2] ipsec: small fixes and switch to basic crypto op type

### PR DESCRIPTION
**Pull request by JannePeltonen**
example: ipsec_crypto: switch to basic crypto operation type
linux-gen: test: fix skipping ipsec examples in process mode
linux-gen: loop: fix use of stale pkt_hdr after inline ipsec
linux-gen: ipsec: switch to basic crypto operation type

### CODE REVIEW

File: example/ipsec_crypto/odp_ipsec.c [hunk 1]
```diff
@@ -634,7 +634,6 @@ pkt_disposition_e do_ipsec_in_classify(odp_packet_t *pkt,
 	/* Initialize parameters block */
 	memset(&params, 0, sizeof(params));
 	params.session = entry->state.session;
-	out_pkt = entry->in_place ? *pkt : ODP_PACKET_INVALID;
```
Comment (MatiasElo) [inline example/ipsec_crypto/odp_ipsec.c:637]: Copyright can be updated.

Comment (JannePeltonen) [reply example/ipsec_crypto/odp_ipsec.c:637]: I'll do that later in a PR that will have more substantial changes to the code.

File: platform/linux-generic/test/example/ipsec_crypto/pktio_env [hunk 2]
```diff
@@ -35,7 +35,7 @@ if [ -n "$WITH_OPENSSL_CRYPTO" ] && [ ${WITH_OPENSSL_CRYPTO} -eq 0 ]; then
 	exit 77
 fi
 
-if [ ${ODPH_PROC_MODE} -eq 1 ]; then
+if [ -n "$ODPH_PROC_MODE" ] && [ ${ODPH_PROC_MODE} -ne 0 ]; then
```
[the remark this answers is not in the document]
Comment (JannePeltonen) [reply platform/linux-generic/test/example/ipsec_crypto/pktio_env:38]: Indeed. Fixed the same way there in v2.

**Comment by JannePeltonen** [comment]
v2: added review tags and fixed pktio_env for ipsec_api example too.

**Pull request merged**